src/Utils/util.ts

```diff
@@ -5,6 +5,7 @@
 import * as CLM from '@conversationlearner/models'
 import * as IntlMessages from '../react-intl-messages'
 import * as moment from 'moment'
+import * as stringify from 'fast-json-stable-stringify'
```

Comment:
As mentioned earlier this needs to be declared as dependency.  

I think you had mentioned some other package already required. We can't depend on this. Always declare what you need, if that package changes and removes their dependency our app would break.